src/actions/index.js

```diff
@@ -235,3 +245,18 @@ export const getInvites = ({ pursuanceId }) => ({
   type: 'GET_INVITES',
   pursuanceId
 });
+
+export const postMembership = membership => ({
+  type: 'POST_MEMBERSHIP',
+  payload: postMembershipReq(membership)
+});
+
+export const getMemberships = filterOption => ({
+  type: 'GET_MEMBERSHIPS',
+  payload: getMembershipsReq(filterOption)
+});
+
+export const delMembership = membership => ({
+  type: 'DELETE_MEMBERSHIP',
+  payload: delMembershipReq(membership)
```

Comment:
@Moe-Shoman Please rename to `deleteMembershipReq`, just to follow the convention we've been using

Reply:
done